package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "stylelint-config-cssrecipes",
+  "version": "1.0.0",
+  "description": "cssrecipes shareable config for stylelint",
+  "keywords": [
+    "browser",
+    "style",
+    "css",
+    "css-components",
+    "css-recipes",
+    "cssrecipes",
+    "recipes",
+    "linter"
```

Comment:
postcss seems to be the exception with the hyphen here:

[eslint](http://eslint.org/docs/developer-guide/working-with-plugins.html#share-plugins), [gulp](https://github.com/gulpjs/gulp/blob/master/docs/writing-a-plugin/guidelines.md) and grunt seem to use single words: `eslintplugin`, `gulpplugin` and `gruntplugin`.

I suggest we stick with "stylelintconfig" for consistency with the majority. But I'm easy either way :)